test/k8sT/manifests/bookinfo-v1-istio.yaml

```diff
@@ -128,17 +128,20 @@ spec:
         resources:
           limits:
             cpu: "2"
-            memory: 128Mi
+            memory: 1Gi
```

Comment:
1Gi? for a proxy? :scream:

Reply:
Yes, this is from the Istio, not me :-) Besides, its the limit, not the minimum :-)